src/crimson/os/seastore/random_block_manager.h

```diff
@@ -22,13 +22,29 @@
 
 namespace crimson::os::seastore {
 
+struct rbm_shard_info_t {
+  std::size_t size = 0;
+  uint64_t start_offset = 0;
+  uint64_t journal_size = 0;
```

Comment:
Is it same to `rbm_metadata_header_t::journal_size` ?

If so, I think we should only keep only one to be persistent.